.github/workflows/images-creator.yml

```
@@ -123,3 +123,29 @@ jobs:
         run: |
           docker push diracgrid/slc6-dirac:latest
           docker push docker.pkg.github.com/diracgrid/management/slc6-dirac:latest
+
+jobs:
+  # Image for deploying config files to CVMFS (GridFTP)
+  dirac-cvmfs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Since you are here, you can use 
```suggestion
      - uses: actions/checkout@v2
```
here, and in the ones above.

---

done